addons/l10n_lt/data/account_tax_group_data.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+    <!-- VAT classificator (lt: 'PVM klasifikatorius') -->
+    <record id="tax_group_vat" model="account.tax.group">
+      <field name="name">VAT</field>
+      <field eval="20" name="sequence"/>
+    </record>
+    <!-- Profit tax classificator (lt: 'Pelno mokesčio klasifikatorius') -->
+    <record id="tax_group_pt" model="account.tax.group">
```

Comment:
I don't see `tax_group_pt` and `tax_group_aap` referred anywhere, can you assign them to appropriate taxes or remove them otherwise

Reply:
Looks like left overs. Will double check, but probably not needed, so will remove it.